index.html

```diff
@@ -278,22 +280,269 @@
+                                    <path fill="#333333" d="M266.86,89.886c0.176,0,0.373-0.01,0.594-0.029c0.219-0.02,0.432-0.044,0.637-0.073s0.393-0.063,0.564-0.103
+                                    	c0.17-0.039,0.3-0.073,0.388-0.103V91.6c-0.127,0.059-0.288,0.117-0.483,0.176s-0.41,0.11-0.645,0.154s-0.479,0.078-0.732,0.103
+                                    	s-0.503,0.037-0.747,0.037c-0.605,0-1.167-0.078-1.685-0.234s-0.965-0.425-1.34-0.806c-0.377-0.381-0.672-0.896-0.887-1.545
+                                    	s-0.322-1.467-0.322-2.454v-9.331h-2.271v-1.187l2.271-1.143L263.345,72h1.538v3.691h4.043v2.007h-4.043v9.331
+                                    	c0,0.947,0.153,1.66,0.462,2.139C265.651,89.646,266.157,89.886,266.86,89.886z"/>
+                                    <path fill="#333333" d="M278.736,75.398c0.283,0,0.601,0.012,0.952,0.037s0.654,0.061,0.908,0.11l-0.352,2.461
+                                    	c-0.273-0.049-0.555-0.09-0.842-0.125c-0.289-0.034-0.56-0.051-0.813-0.051c-0.615,0-1.177,0.125-1.685,0.374
+                                    	s-0.945,0.605-1.311,1.069c-0.367,0.464-0.65,1.021-0.85,1.67c-0.201,0.649-0.301,1.37-0.301,2.161v8.672h-2.666V75.691h2.168
+                                    	l0.322,2.944h0.117c0.244-0.42,0.506-0.825,0.783-1.216c0.279-0.391,0.591-0.735,0.938-1.033s0.735-0.537,1.165-0.718
+                                    	S278.189,75.398,278.736,75.398z"/>
+                                    <path fill="#333333" d="M289.36,92.068c-1.074,0-2.056-0.181-2.944-0.542s-1.653-0.889-2.292-1.582
+                                    	c-0.641-0.693-1.136-1.553-1.487-2.578s-0.527-2.197-0.527-3.516c0-1.328,0.161-2.515,0.483-3.56s0.776-1.931,1.362-2.659
+                                    	s1.286-1.282,2.103-1.663c0.814-0.381,1.716-0.571,2.702-0.571c0.967,0,1.836,0.173,2.607,0.52s1.426,0.835,1.963,1.465
+                                    	s0.949,1.387,1.237,2.271S295,81.517,295,82.591v1.655h-10.137c0.049,1.885,0.446,3.271,1.194,4.16
+                                    	c0.746,0.889,1.857,1.333,3.332,1.333c0.498,0,0.959-0.024,1.385-0.073c0.424-0.049,0.834-0.12,1.23-0.212
+                                    	c0.395-0.093,0.778-0.21,1.149-0.352s0.747-0.3,1.128-0.476v2.373c-0.391,0.186-0.774,0.347-1.15,0.483s-0.762,0.249-1.156,0.337
+                                    	c-0.396,0.088-0.807,0.151-1.23,0.19C290.319,92.049,289.858,92.068,289.36,92.068z M288.701,77.61
+                                    	c-1.113,0-1.995,0.381-2.644,1.143c-0.65,0.762-1.028,1.87-1.136,3.325h7.207c0-0.664-0.065-1.272-0.198-1.824
+                                    	c-0.131-0.552-0.336-1.023-0.615-1.414c-0.277-0.391-0.632-0.693-1.062-0.908S289.307,77.61,288.701,77.61z"/>
+                                    <use xlink:href="#idrc_symbol"  width="116" height="148" x="-58" y="-74" transform="matrix(0.5 0 0 -0.5 54 60)" overflow="visible"/>
+                                    </svg>
```

Comment:
Could you adjust the indentation so that the contents of the svgs is indented one level.